Mathlib/Analysis/Distribution/Distribution.lean

```diff
@@ -0,0 +1,185 @@
+/-
+Copyright (c) 2025 Anatole Dedecker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anatole Dedecker
+-/
+module
+
+public import Mathlib.Analysis.Distribution.TestFunction
+public import Mathlib.Analysis.LocallyConvex.StrongTopology
+
+/-!
+# Distributions
+
+Let `E` be a real **finite-dimensional normed space**, `Ω` an open subset of `E`,
+and `F` a real **locally convex topological vector space**.
+
+An **`F`-valued distribution on `Ω`** is a continuous `ℝ`-linear map `T : 𝓓(Ω, ℝ) →L[ℝ] F`,
+defined on the space `𝓓(Ω, ℝ)` of real-valued test functions, and taking values in `F`.
+In particular, if `𝕜` is an `RCLike` field, `𝓓'(Ω, 𝕜)` is the usual notion of real or complex
+distribution on `Ω`.
+
+We denote the space of `F`-valued distributions on `Ω` by `𝓓'(Ω, F)`. Topologically,
```

Comment:
I agree that `𝓓'` is the classical choice (although Schwartz even add parentesis and write `(𝓓')`...), but given that it is decorated with many other variables here (including possibly the exponent `{n}`), I think that I would prefer `Distr(Ω, 𝕜)` and `Distr^{n}(Ω,𝕜)`. These are five characters to type, and `𝓓` are at any rate four: \MCD.